addons/point_of_sale/models/pos_order.py

```diff
@@ -282,7 +282,7 @@ def _compute_currency_rate(self):
     @api.onchange('payment_ids', 'lines')
     def _onchange_amount_all(self):
         for order in self:
-            currency = order.pricelist_id.currency_id
+            currency = order.pricelist_id.currency_id or self.env.user.company_id.currency_id
```

Comment:
better use the currency of the pos_config linked
```suggestion
            currency = order.pricelist_id.currency_id or order.config_id.currency_id
```

Reply:
I just tried it and the order doesn't contain any `config_id` when using studio so the currency_id is still empty